tests/serverpod_test_server/test_integration/database_operations/transactions/transaction_isolatation_level_test.dart

```diff
@@ -0,0 +1,292 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
```

Comment:
Question: Why couldn't we use our test tools but and disable the database rollbacks?

Reply:
I think it makes sense to do the switch so that we're moving toward the `withServerpod` test paradigm (even though it will be a long migration process). So in this case you would have to add the following and it will work:
```dart
import 'package:serverpod_test_server/test_util/test_tags.dart';
// ...
rollbackDatabase: RollbackDatabase.disabled,
testGroupTagsOverride: [concurrencyOneTestTag],
```

Reply:
(as a bonus, acutally I think we should do like i did in cloud and declare the test tag like this):
```dart
abstract final class TestTags {
  concurrencyOne = 'concurrencyOne';
}
```

Reply:
60a9018f02e19879afb9fddbceda5d686c4767eb and 2165a9d334f91ba32fa4cb112a9565eb8025ffc6